utils/api.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios'
+
+const newsAPI = axios.create({
+    baseURL: "https://nc-news-api-fa1t.onrender.com/api"
+})
+
+export function getAllArticles() {
+    return newsAPI.get("/articles");
```

Comment:
It may be a good idea to take the data off the relevant keys in this function rather than it cluttering up your component